src/config.py

```diff
@@ -1,14 +1,24 @@
 """In-app config. Set by environment variables."""
 
 import os
-from typing import Set
+from typing import Set, Optional
 import re
 from dotenv import load_dotenv, find_dotenv
 
 load_dotenv(find_dotenv())
 
-SBERT_MODEL: str = os.getenv("SBERT_MODEL", "msmarco-distilbert-dot-v5")
-INDEX_ENCODER_CACHE_FOLDER: str = os.getenv("INDEX_ENCODER_CACHE_FOLDER", "/models")
+SBERT_MODELS: list[str] = [
+    "BAAI/bge-small-en-v1.5",  # 384 dim
+    "BAAI/bge-base-en-v1.5",  # 768 dim
+    "msmarco-distilbert-base-tas-b",  # 768 dim
+]
```

Comment:
You're using the dimension later in a test, so might be worth creating a map here, rather than just commenting the dimensions

Reply:
agree but will leave for now as the dimensions are just used for tests